src/js/pages/ManualPurchaseModal.js

```diff
@@ -0,0 +1,48 @@
+import Component from "../lib/Component.js";
+import {$, $$} from "../modules/utils.js";
+
+export default class ManualPurchaseModal extends Component {
+  template() {
+    const {isManualModalVisible} = this.props;
+    return `
+      <div class="modal${isManualModalVisible ? ' open' : ''}">
+        <div class="modal-inner p-10">
+          <div id="manualModalClose" class="modal-close">
+            <svg viewbox="0 0 40 40">
+              <path class="close-x" d="M 10,10 L 30,30 M 30,10 L 10,30" />
+            </svg>
+          </div>
+          <h2>로또 수동구매</h2>
+          <div class="d-flex justify-center">
+            <section class="mt-9">
+              <div class="d-flex">
+                <div>
+                  <h4 class="mt-0 mb-3 text-center">로또번호 6개를 입력해주세요.</h4>
+                  <div>
+                    ${
+                      new Array(6).fill().reduce((str, ele, i) => {
```

Comment:
```suggestion
                      new Array(6).fill().reduce((str) => {
```
사용하지 않는 파라미터는 지워주세요!